tests/runtime/test_device_qir.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (C) 2024 qBraid
+#
+# This file is part of the qBraid-SDK
+#
+# The qBraid-SDK is free software released under the GNU General Public License v3
+# or later. You can redistribute and/or modify it under the terms of the GPL v3.
+# See the LICENSE file in the project root or <https://www.gnu.org/licenses/gpl-3.0.html>.
+#
+# THERE IS NO WARRANTY for the qBraid-SDK, as per Section 15 of the GPL v3.
+
+# pylint: disable=redefined-outer-name,unused-argument,too-many-lines
+
+"""
+Unit tests for QbraidDevice, QbraidJob, and QbraidGateModelResultBuilder
+classes using the qbraid_qir_simulator
+
+"""
+import pytest
+pytest.importorskip("pyqir")
+from pyqir import BasicQisBuilder, Module, SimpleModule
```

Comment:
pytest importskip returns module same as above

Reply:
since python doesn't have dep. types but we use the Module type for example in the rest of the file, I had to import Module and others at some point as opposed to taking the `pyqir` module from `importorskip` and using `pyqir.Module`